### PR TITLE
Update readme and docs

### PR DESCRIPTION
**Pull request by nipunj15**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,96 @@
...
-Try running the following commands:
-- dbt run
-- dbt test
+This package enables you to access commonly used metrics on top of Stripe transactions.
 
+## Metrics 
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
+This package contains tranformed models built on top of [Fivetran Stripe source package](https://github.com/fivetran/dbt_stripe_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The metrics offered by this package are described below
```
Comment (sp35) [inline README.md]: typo `tranformed`

File: README.md [hunk 2]
```diff
@@ -1,15 +1,96 @@
...
+| [stripe__mrr](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__mrr.sql)    | Monthly recurring revenue.
+| [stripe__new_customer_revenue_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__new_customers_monthly.sql)    |         Monthly revenue due to customers that signed up for the first time.                                                               |
+| [stripe__new_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__churned_customers_monthly.sql)    |  Monthly count of customers that signed up for the first time.                                     |
+| [stripe__platform_fees_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__platform_fees_monthly.sql)    | Monthly fees paid to Stripe.                                                         |
+| [stripe__recovered_customer_revenue_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__recovered_customer_revenue_monthly.sql)    | Monthly revenue due recovery of previously churned customers.               |
```
Comment (sp35) [inline README.md]: Monthly revenue due to

File: README.md [hunk 3]
```diff
@@ -1,15 +1,96 @@
...
+
+```yaml
+packages:
+  - git: "git@github.com:HousewareHQ/dbt_stripe_metrics.git"
+    revision: 0.1.0
+```
+
+## Step 3: Define database and schema variables
+
+By default, this package will look for your Stripe data in the `stripe` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Stripe data is, please add the following configuration to your `dbt_project.yml` file:
```
Comment (sp35) [inline README.md:45]: Let's either update the variable `stripe_schema` in dbt_project.yml to `stripe` or the update here as `fivetran_stripe`

File: README.md [hunk 4]
```diff
@@ -1,15 +1,96 @@
...
+vars:
+  stripe_source:
+    stripe_database: your_database_name
+    stripe_schema: your_schema_name
+```
+
+For additional configurations for the source models, please visit the [Stripe source package](https://github.com/fivetran/dbt_stripe_source).
+
+## (Optional) Step 4: Change build schema
+By default this package will build the Stripe staging models within a schema titled (<target_schema> + `_stg_stripe`) and the Stripe metrics within a schema titled (<target_schema> + `_stripe_metrics`) in your target database. If this is not where you would like your modeled Stripe data to be written to, add the following configuration to your `dbt_project.yml` file:
```
Comment (sp35) [inline README.md:62]: Let's update default target schema to `stg_stripe` in dbt_project.yml

File: README.md [hunk 5]
```diff
@@ -1,15 +1,96 @@
...
+...
+models:
+  stripe_metrics:
+    +schema: my_new_schema_name # leave blank for just the target_schema
+  stripe_source:
+    +schema: my_new_schema_name # leave blank for just the target_schema
+```
+
+
+# 🗄 Which databases are supported?
```
Comment (sp35) [inline README.md]: warehouse or database ?

File: README.md [hunk 6]
```diff
@@ -1,15 +1,96 @@
...
+
+
+# 🏪 Are there any resources available?
+- Provide [feedback](https://airtable.com/shrPHxTmfkjq3P6Eh) on what you'd like to see next
+- Have questions, feedback, or need help? Email us at nipun@houseware.io
+- Check out [Houseware's blog](https://www.houseware.io/blog)
+- Learn more about dbt [in the dbt docs](https://docs.getdbt.com/docs/introduction)
 - Check out [Discourse](https://discourse.getdbt.com/) for commonly asked questions and answers
-- Join the [chat](https://community.getdbt.com/) on Slack for live discussions and support
+- Join the [chat](http://slack.getdbt.com/) on Slack for live discussions and support
```
Comment (sp35) [inline README.md]: https 😅

File: models/schema.yml [hunk 7]
```diff
@@ -204,39 +204,39 @@ models:
...
+    description: "Monthly fees paid to Stripe."
     columns:
       - name: period
         description: ""
       - name: stripe__platform_fees_monthly
         description: ""
 
   - name: stripe__recovered_customer_revenue_monthly
-    description: ""
+    description: "Monthly revenue due recovery of previously churned customers."
```
Comment (sp35) [inline models/schema.yml:231]: due to

File: README.md (hunk 4 above)
Comment (sp35) [reply README.md:62]: or we can remove this configuration to rely on `fivetran_source` default

File: README.md (hunk 3 above)
Comment (sp35) [reply README.md:45]: https://github.com/HousewareHQ/dbt_stripe_metrics/pull/5#discussion_r905977780

File: README.md (hunk 4 above)
Comment (nipunj15) [reply README.md:62]: `stg_stripe` is the default in fivetran `stripe_source` package

**Review by sp35 — APPROVED** [review]

**Pull request merged**